### PR TITLE
docs: 5G SA Feature Documentation

### PR DESCRIPTION
**Pull request by panyogesh**
Signed-off-by: Yogesh Pandey <yogesh@wavelabs.ai>

## Summary

Documents for Integrated 5G SA

## Test Plan

Integrated-5G-SA-Snapshots-1
Integrated-5G-SA-Snapshots-2
Integrated-5G-SA-Snapshots-3

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

File: docs/readmes/lte/5G-sa-document.md [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+
+This document gives an overview of the 5G SA functionality present in Magma, its architecture, configuration, feature matrix
+and general debugging techniques.
+
+## Architecture
+
+Currently, fixed wireless access deployment is targeted as the first milestone for 5G SA.
+
+AMF included in current MME module and will serve as entry point for all the N1 and N2 messages coming from CPE and GNB.
+Post processing of these messages AMF will connect with existing subscriberdb, mobilityd and sessiond module for realising the end to end call flow.
```
[the remark this answers is not in the document]
Comment (panyogesh) [reply docs/readmes/lte/5G-sa-document.md]: Done.

File: docs/docusaurus/i18n/en.json [hunk 2]
```diff
@@ -63,14 +63,14 @@
...
       },
-      "dp/debug_logs": {
-        "title": "View Logs"
+      "dp/debug_logs title: View Logs hide_title: true": {
+        "title": "dp/debug_logs title: View Logs hide_title: true"
       },
-      "dp/deploy_build": {
-        "title": "Build DP"
+      "dp/deploy_build title: Build DP hide_title: true": {
+        "title": "dp/deploy_build title: Build DP hide_title: true"
```
Comment (uri200) [inline docs/docusaurus/i18n/en.json]: why those changes on DP?

**Comment by uri200** [comment]
Thanks @panyogesh for this detailed documentation. 
Mermaid graph really help to ease future maintenance!

**Comment by uri200** [comment]
As a test plan, can you run 
```
docs/docusaurus/create_docusaurus_website.sh
```
To build locally docusaurus and add a screen shoot showing the new document please?

File: docs/docusaurus/i18n/en.json (hunk 2 above)
Comment (panyogesh) [reply docs/docusaurus/i18n/en.json]: Till "make precommit" it was fine showing only 2 line change.
Other changes started showing when executed "make dev". Should I remove it ?

Comment (uri200) [reply docs/docusaurus/i18n/en.json]: Oh that code is autogenerated. So probably it was not committed or the generator has been updated and now is generating some different strings.

It is fine, you can leave it as is.

**Comment by shanku9** [comment]
@panyogesh This configuration requires orchestrator and swagger API access 
this might be challenging for shared tenant orc8r setup as well as for testing agw in headless mode. Can we also publish local override files?
Can we also add 5G feature verification guide for the end user? 
::::::::::::::::::::::::::::
Check sctp port 38412
cat /proc/net/sctp/eps
ENDPT     SOCK   STY SST HBKT LPORT   UID INODE LADDRS
ffff8b5fb7b99500 ffff8b5fbf1f9200 2   10  21   36412     0 702712428 10.22.0.160
ffff8b5dbdd80d00 ffff8b5fb4356000 2   10  37   38412     0 702714036 10.22.0.160
::::::::::::::::::::::::::::

**Comment by ekowtaylor** [comment]
Since this is in progress, let's try to see it through so it gets merged

**Comment by panyogesh** [comment]
Integrated-5G-SA-Snapshots-4
Integrated-5G-SA-Snapshots-5

**Comment by panyogesh** [comment]
> @panyogesh This configuration requires orchestrator and swagger API access this might be challenging for shared tenant orc8r setup as well as for testing agw in headless mode. Can we also publish local override files? Can we also add 5G feature verification guide for the end user? :::::::::::::::::::::::::::: Check sctp port 38412 cat /proc/net/sctp/eps ENDPT SOCK STY SST HBKT LPORT UID INODE LADDRS ffff8b5fb7b99500 ffff8b5fbf1f9200 2 10 21 36412 0 702712428 10.22.0.160 ffff8b5dbdd80d00 ffff8b5fb4356000 2 10 37 38412 0 702714036 10.22.0.160 ::::::::::::::::::::::::::::

Comments addressed.

**Review by uri200 — APPROVED** [review]

**Review by themarwhal — APPROVED** [review]

**Pull request merged**